libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/components/CodeLanguageSelector.tsx

```diff
@@ -0,0 +1,55 @@
+// (C) 2022 GoodData Corporation
+import React, { ChangeEvent, useCallback } from "react";
+import { FormattedMessage } from "react-intl";
+import { CodeLanguageType } from "../types";
+
+/**
+ * @internal
+ */
+export interface ICodeLanguageSelectorProps {
+    selectedLanguage: CodeLanguageType;
+    onLanguageChanged: (language: CodeLanguageType) => void;
+}
+
+/**
+ * @internal
+ */
+export const CodeLanguageSelector: React.VFC<ICodeLanguageSelectorProps> = (props) => {
```

Comment:
I don't know if `Selector` is the right word, what about `CodeLanguageSelect` (we already use Selector suffix for redux selectors)?

Reply:
Done